CHANGELOG

```diff
@@ -1,3 +1,12 @@
+Release 162 (2021-06-22)
+------------------------
+Add com.vimeo/interactive_overlay_panel_click_event/jsonschema/1-0-0 (close #1307)
+Add com.vimeo/interactive_hotspot_click_event/jsonschema/1-0-0 (close #1306)
+Add com.vimeo/interaction/jsonschema/1-0-0 (close #1297)
+Add com.vimeo/chapter_change_event/jsonschema/1-0-0 (close #1280)
+Add com.vimeo/cue_point_event/jsonschema/1-0-0 (close #1281)
```

Comment:
There is a missing line for the `meta` schema you have added @greg-el.  Can you please add that to the CHANGELOG and then ill merge it in!

Reply:
Added!